### PR TITLE
Feat/add embed mentions argument in from message method

### PR DESCRIPTION
**Pull request by Kondrahin**
# Description

Add `embed_mentions` argument in `from_message` method.
Fix `EMBED_MENTION_RE` expression

# Checklist

- [x] Linters and tests have passed
- [x] I've written a changelog for PR change
- [x] I'll make a release when PR is approved

### CODE REVIEW

**Review by lxmnk — APPROVED** [review]

**Review by Nimond — APPROVED** [review]

File: botx/models/messages/sending/message.py [hunk 1]
```diff
@@ -24,13 +24,14 @@
...
+EMBED_MENTION_TEMPLATE = (
     "<embed_mention:{mention_type}:"
     "{mentioned_entity_id}:{mention_id}:{mention_name}>"  # noqa: WPS326
 )
 EMBED_MENTION_RE: Final = re.compile(
-    "<embed_mention:(?P<mention_type>.+?):(?P<mentioned_entity_id>.+?)"
-    ":(?P<mention_id>.+?):(?P<name>.+?)?>",  # noqa: WPS326 C812
+    "<embed_mention:(?P<mention_type>.+?):"
+    r"(?P<mentioned_entity_id>[0-9a-f\-]+?):"
+    r"(?P<mention_id>[0-9a-f\-]+?):(?P<name>.+?)??>"  # noqa: WPS326 C812
```
Comment (vonabarak) [inline botx/models/messages/sending/message.py:34]: May be need to add some unit-tests checking this regexp works correctly for each case. 
E.g. with optional <name> group and without it.

File: botx/models/messages/sending/message.py [hunk 2]
```diff
@@ -24,13 +24,14 @@
...
 
-EMBED_MENTION_TEMPATE = (
+EMBED_MENTION_TEMPLATE = (
     "<embed_mention:{mention_type}:"
     "{mentioned_entity_id}:{mention_id}:{mention_name}>"  # noqa: WPS326
 )
 EMBED_MENTION_RE: Final = re.compile(
-    "<embed_mention:(?P<mention_type>.+?):(?P<mentioned_entity_id>.+?)"
-    ":(?P<mention_id>.+?):(?P<name>.+?)?>",  # noqa: WPS326 C812
+    "<embed_mention:(?P<mention_type>.+?):"
```
Comment (vonabarak) [inline botx/models/messages/sending/message.py:32]:
```suggestion
    "<embed_mention:(?P<mention_type>user|contact|chat|channel|all):"
```

Comment (lxmnk) [reply botx/models/messages/sending/message.py:32]: I think it's overspecification. This check already [exists](https://github.com/ExpressApp/pybotx/blob/71eedabfb10a88de82a3b62ac669315b9e25c9dd/botx/models/messages/sending/message.py#L528).

Comment (vonabarak) [reply botx/models/messages/sending/message.py:32]: Ok. So let's add unit-test, that checks wrong mention type will raise an exception.

**Review by vonabarak — APPROVED** [review]

**Pull request merged**